Code/Mantid/MantidQt/MantidWidgets/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ set ( SRC_FILES
     src/MuonFitPropertyBrowser.cpp
     src/MuonSequentialFitDialog.cpp
     src/PeakPicker.cpp
+	src/PeriodicTableWidget.cpp
     src/PreviewPlot.cpp
```

Comment:
it looks like there's a tab/spaces issue here. I'd try to keep all the lines with identical indentation, as many of our CMakeLists.txt have a tendency to become messy because of indentation issues.